spec/prog/vm/github_runner_spec.rb

```diff
@@ -23,7 +23,7 @@
   let(:installation) { runner.installation }
   let(:project) { installation.project }
   let(:client) { instance_double(Octokit::Client) }
-  let(:now) { Time.utc(2025, 5, 19, 19, 0) }
+  let(:now) { Time.utc(2025, 8, 1, 19, 0) }
```

Comment:
Is this change necessary? No objections to it.

Reply:
Otherwise, it can't find the active billing rates. It should be later than the `active_from` date.